### PR TITLE
introduce max_message_size config option

### LINKED ISSUE #268: Two more variables into the chatmail.ini
As chatmail is recommended for small groups, the compliance into this group can it make affordable to change the maximal size of the message and the time to delete an inactive account, too. 

Please add these two variables into chatmail.ini if it is possible:

max_size =              # maximal size for a message in megabyte
time_to_delete =    # an inactive acoount in days

Thank you!

### PR DESCRIPTION
**Pull request by hpk42**
fixes #268 and also fixes a format-string typo/error on the side.

### CODE REVIEW

File: chatmaild/src/chatmaild/config.py [hunk 1]
```diff
@@ -73,7 +74,7 @@ def write_initial_config(inipath, mail_domain, overrides):
         new_line = line.strip()
         if new_line and new_line[0] not in "#[":
             name, value = map(str.strip, new_line.split("=", maxsplit=1))
-            value = overrides.pop(name, value)
+            value = extra.pop(name, value)
```
Comment (hpk42) [inline chatmaild/src/chatmaild/config.py:77]: this is an unrelated fix which was needed for the added test.

File: cmdeploy/src/cmdeploy/dns.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ def get_initial_remote_data(sshexec, mail_domain):
 def check_initial_remote_data(remote_data, print=print):
     mail_domain = remote_data["mail_domain"]
     if not remote_data["A"] and not remote_data["AAAA"]:
-        print("Missing A and/or AAAA DNS records for {mail_domain}!")
+        print(f"Missing A and/or AAAA DNS records for {mail_domain}!")
```
Comment (hpk42) [inline cmdeploy/src/cmdeploy/dns.py:18]: this is also unrelated but doing an extra PR is too much.

**Review by missytake — APPROVED** [review]
